spec.html

```diff
@@ -23638,7 +23637,7 @@
     <!-- es6num="19.5.3" -->
     <emu-clause id="sec-properties-of-the-error-prototype-object">
       <h1>Properties of the Error Prototype Object</h1>
-      <p>The Error prototype object is the intrinsic object <dfn>%ErrorPrototype%</dfn>. The Error prototype object is an ordinary object. It is not an Error instance and does not have an [[ErrorData]] internal slot.</p>
+      <p>The Error prototype object is the intrinsic object <dfn>%ErrorPrototype%</dfn>. The Error prototype object is an ordinary object. It is not an Error instance.</p>
```

Comment:
I think it makes as much sense as any of the other "Is not an X instance" statements. Presumably they are all combating the same misconception from ES3 days that the prototype is some kind of exemplar of the type. I don't see why this change would impact things.